geode-core/src/main/java/org/apache/geode/management/CacheServerMXBean.java

```diff
@@ -60,48 +61,48 @@
   /**
    * Returns the port on which this CacheServer listens for clients.
    */
-  public int getPort();
+  int getPort();
```

Comment:
I think this should be ok actually, since CacheServerMXBean is an interface and not a class.  (So all methods will be public by default.)